File: sentimental-mario-less/mario.py
from cs50 import get_int

height = get_int("Height: ")

while height < 1 or height > 8:
    height = get_int("Height: ")


def print_row(lenght):
    for i in range(lenght):
        print('#', end="")


def print_void(lenght):
    for i in range(lenght):
        print(" ", end="")


for i in range(height):
    print_void(height - i - 1)
    print_row(i + 1)
    print("")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int length);
void print_void(int length);

int main(void)
{
    int length;
    do
    {
        length = get_int("how long you want pyramid? ");
    }
    while (!(length > 0));
    for (int i = 0; i < length; i++)
    {
        print_void(length - i - 1);
        print_row(i + 1);
    }
}

void print_row(int length)
{
    for (int i = 0; i < length; i++)
    {
        printf("#");
    }
    printf("\n");
}

void print_void(int length)
{
    for (int i = 0; i < length; i++)
    {
        printf(" ");
    }
}
